pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/bestpractices/UnusedPrivateFieldRule.java

```diff
@@ -43,16 +46,26 @@ public class UnusedPrivateFieldRule extends AbstractJavaRule {
                 + "This property has been deprecated since PMD 6.50.0 and will be completely ignored.")
             .defaultValue(new ArrayList<String>())
             .build();
+
     private static boolean warnedAboutDeprecatedIgnoredAnnotationsProperty = false;
     private static final PropertyDescriptor<List<String>> IGNORED_FIELD_NAMES =
                 PropertyFactory.stringListProperty("ignoredFieldNames")
                     .defaultValues("serialVersionUID", "serialPersistentFields")
                     .desc("Field Names that are ignored from the unused check")
                     .build();
 
+    private static final PropertyDescriptor<List<String>> ANNOTATIONS_DESCRIPTOR
+            = stringListProperty("annotations")
+            .desc("Fully qualified names of the annotation types that should be reported anyway. If an unused field "
+                    + "has any of these annotations, then it is reported. If it has any other annotation, then "
+                    + "it is still considered to used and is not reported.")
+            .defaultValue(Arrays.asList("org.openqa.selenium.support.FindBy"))
```

Comment:
It has pros and cons:

Pro for adding it:
- PMD works out of the box for most (what ever this is, maybe 80%?) of the use cases without the need to tweak it.

Con for adding it:
- what you said - we might accumulate annotations from every framework. Switching from "ignoreAnnotations" to "reportForAnnotations" hopefully reduces the probability for this problem - but we don't know.
- we don't educate users to actually configure the rules they use because PMD simply works
- we don't have clear guidelines established, when - like in this case - an annotation/framework is common enough to be considered in the default values of rules.

I'll keep it empty for now - because adding a default value can be done at any time later, if there are enough supporters from the community, who desperately want to to be the default.